addons/mail/static/src/owl/components/composer_text_input/composer_text_input.scss

```diff
@@ -5,35 +5,14 @@
 .o_ComposerTextInput {
     min-width: 0;
 
-    &.o-mobile > .note-editor > .note-editing-area {
-        padding: 15px 10px 10px 10px;
-    }
-
-    > .note-editor {
-        height: 100%;
-
-        > .note-editing-area {
-            overflow: auto;
-            height: 100%;
-
-            > .note-editable[contenteditable=true] {
-                max-height: 200px;
-                height: 100%;
-            }
-
-            > .note-placeholder {
-                padding: 0;
-            }
-        }
-
-        > .note-statusbar {
-            display: none;
-        }
-
-        > .note-toolbar {
-            border: 0;
-            padding: 0;
-        }
+    > textarea {
```

Comment:
put class `o_ComposerTextInput_textarea` on `textarea`, then makes this style top-level (no scss hierarchy)

Reply:
Done